include/aspect/material_model/interface.h

```diff
@@ -693,6 +681,24 @@ namespace aspect
         /**
          * @}
          */
+
+      protected:
+        /**
+         * A structure that describes how each of the model's
+         * output variables (such as viscosity, density, etc) depend
+         * on the input variables pressure, temperature, strain rate,
+         * and compositional fields.
+         *
+         * The constructor of this class calls the default
+         * constructor of this member variable which in turn
+         * initializes the object to invalid values. Derived classes
+         * then need to fill it either in their constructor (if they
+         * already know the correct dependences at that time) or
+         * at the end of their parse_parameter() functions where
```

Comment:
Does it need to be the parse_parameters() function? I would think, this is a case for the initialize() function.